File: Data Structures & Algorithms/concatenation-of-array/submission-2.cpp
// class Solution{

//     public: vector<int> getConcatenation(vector <int> &nums){
        
//         int n = nums.size();
//         vector <int> ans(n*2);

//         for(int i=0; i<n; i++){

//             ans[i] = nums[i];
//             ans[i+n] = nums[i];

//         }

//         return ans;



//     }

// };


class Solution{

public: vector<int> getConcatenation(vector<int> &nums){
    vector <int>& ans = nums;
    int n = nums.size();

    for(int i =0; i<n; i++){

        ans.push_back(nums[i]);

    }

    return ans;
}

};

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.py
import heapq
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:

        nums = [-i for i in nums]
        heapq.heapify(nums)

        count = 0

        while count <k:
            ans = -heapq.heappop(nums)
            count+=1

        return ans
        